model/task/task.go

```diff
@@ -1710,8 +1710,7 @@ func (t *Task) Archive() error {
 	archiveTask.Archived = true
 	err := db.Insert(OldCollection, &archiveTask)
 	if err != nil {
-		grip.Debug(message.WrapError(err, message.Fields{
-			"ticket":          "EVG-12942",
+		grip.Error(message.WrapError(err, message.Fields{
 			"archive_task_id": archiveTask.Id,
 			"old_task_id":     archiveTask.OldTaskId,
```

Comment:
Is it worth keeping this log now? I guess if we ever run into archive errors again  it will be useful but you could say that about most errors.

Reply:
I think so because it give more context than the returned error